city_scrapers/spiders/det_transportation.py

```diff
@@ -0,0 +1,16 @@
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.spiders import CityScrapersSpider
+
+from city_scrapers.mixins import DetCityMixin
+
+
+class DetTransportationSpider(DetCityMixin, CityScrapersSpider):
+    name = "det_transportation"
+    agency = "Detroit Department of Transportation"
+    dept_cal_id = '166'
+    agency_cal_id = 'All'
+    dept_doc_id = '166'
+    agency_doc_id = 'None'
+
+    def _parse_classification(self, response):
```

Comment:
You can override this to return `FORUM` when it's a Community Input meeting, `ADVISORY_COMMITTEE` when it's an Advisory Council meeting and then it can default to `BOARD`.